src/bin/cql-stress-cassandra-stress/operation/mod.rs

```diff
@@ -16,6 +23,121 @@ use scylla::{
 };
 pub use write::WriteOperationFactory;
 
+use crate::settings::CassandraStressSettings;
+use crate::stats::ShardedStats;
+
+use self::row_generator::RowGenerator;
+
+/// A specific CassandraStress operation.
+#[trait_variant::make(CassandraStressOperation: Send)]
+pub trait LocalCassandraStressOperation: Sync + Send + Sized {
+    type Factory: CassandraStressOperationFactory<Operation = Self>;
+
+    const DEFAULT_TABLE_NAME: &'static str = "standard1";
+    const DEFAULT_COUNTER_TABLE_NAME: &'static str = "counter1";
+
+    fn get_stressed_table_name() -> &'static str;
+    async fn execute(&self, row: &[CqlValue]) -> Result<ControlFlow<()>>;
```

Comment:
Please document how this trait interacts with `GenericCassandraStressOperation` - most importantly, how `CassandraStressOperation::execute`'s return value should be interpreted.